### DISCUSSION TITLE
Is portmapper necessary to run longhorn?

### DISCUSSION THREAD

**Discussion by pcellix** [longhorn/longhorn#6237] [Q&A]
Hi,

We want to use longhorn on production with our kubernetes cluster. After setting it up our service provider started sending regarding possible abuse due to portmapper service running
Can it be disabled without affecting longhorn service running?

Best regards,

**Comment by derekbit** [comment]
I don't remember portmapper is used in Longhorn.
@shuo-wu @PhanLe1010 do you know that?

**Reply by shuo-wu** [reply]
IIRC, Longhorn does not use it...

**Comment by pcellix** [comment]
Isn't it used by nfs?

śr., 5 lip 2023 o 05:01 Shuo Wu ***@***.***> napisał(a):

**Reply by derekbit** [reply]
IIRC, no. You can try to disable it.
